core/extraction/towns_fund_round_four.py

```diff
@@ -92,3 +97,32 @@ def extract_programme_progress(df_data: pd.DataFrame, programme_id: str) -> pd.D
     df_data = r3.extract_programme_progress(df_data, programme_id)
     df_data = df_data.drop(df_data.iloc[5].name)  # Question 6 isn't required for Round 4
     return df_data
+
+
+def extract_programme_management(df_data: pd.DataFrame, programme_id: str) -> pd.DataFrame:
+    # if programme_id.split("-")[0] != "TD":  # Return an empty DataFrame if the programme ID is not Town Deal
+    #     return pd.DataFrame(columns=[
+    #         "Programme ID", "Payment Type", "Reporting Period", "Spend for Reporting Period"
+    #     ])
```

Comment:
Looks like this should be uncommented I think? Would be good to also have a test that checks we return an empty df with the correct headers if the fund type is not TD